exercises/accumulate/README.md

```diff
@@ -39,6 +39,36 @@ To raise a message with an exception, just write it as an argument to the except
 raise Exception("Meaningful message indicating the source of the error")
 ```
 
+## Running the tests
+
+To run the tests, run the appropriate command below:
+
+| Python Version | Command |
+| --- | --- |
+| 2.7 | `py.test accumulate_test.py` |
+| 3.3+ | `pytest accumulate_test.py` |
+| 2.7, 3.3+ | `python -m pytest accumulate_test.py` |
```

Comment:
This might be obvious to people who use python, but I have no idea from looking at this why 2.7 and 3.3+ both have two different commands. When would I use one or the other? What would I actually type on my command-line (given that I have 2.7) to run the tests?